terracumber_config/tf_files/SUSEManager-5.0-alternate-base-OS-NUE.tf

```diff
@@ -0,0 +1,220 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-5.0/job/manager-5.0-qe-sle-update-NUE"
```

Comment:
Maybe we should rename this to differentiate from the other? This already exists

Reply:
Done.